compositor_pipeline/src/error.rs

```diff
@@ -59,6 +62,15 @@ pub enum OutputInitError {
     OutputError(#[from] Box<dyn std::error::Error + Send + Sync + 'static>),
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum InputInitError {
+    #[error(transparent)]
+    FfmpegError(#[from] ffmpeg_next::Error),
+
+    #[error(transparent)]
+    InputError(#[from] Box<dyn std::error::Error + Send + Sync + 'static>),
```

Comment:
I'm fine with leaving it as Box<dyn Error>, but if we are implementing automatic From it's too risky. Add some wrapper-type or remove #[from] and map this error manually. (I slightly prefer the second option, but it's not a strong opinion)

The same applies to the encoder

Reply:
makes sense

Reply:
I prefer the second option too, but if I mark the box as `#[source]` instead of `#[from]`, the error cannot be `transparent` :(

I guess we have to stick with the wrapper type